src/components/reusable/timepicker/timepicker.scss

```diff
@@ -0,0 +1,160 @@
+@use '../../../common/scss/global.scss';
+@use '~@kyndryl-design-system/foundation/scss/mixins/typography.scss';
+
+:host {
+  display: inline-block;
+}
+
+.time-picker {
+  width: 147px;
+}
+
+.label-text {
+  @include typography.type-ui-03;
+  text-transform: uppercase;
+  font-weight: 500;
+  display: block;
+  padding: 0;
+  margin-bottom: 8px;
+
+  [disabled] & {
+    color: var(--kd-color-text-disabled);
+  }
+}
+
+.required {
+  color: var(--kd-color-text-error);
+
+  [disabled] & {
+    color: var(--kd-color-text-disabled);
+  }
+}
+
+.input-wrapper {
+  position: relative;
+}
+
+slot[name='icon']::slotted(*) {
+  display: block;
+}
```

Comment:
I don't think there's an icon slot for this component, right?

Reply:
yup will remove this one

Reply:
done